packages/terra-button-group/package.json

```diff
@@ -45,6 +45,6 @@
     "props-table": "props-table ./src/ButtonGroup.jsx ./src/ButtonGroupButton.jsx --out-dir ./docs/props-table",
     "test": "npm run test:jest && npm run test:nightwatch",
```

Comment:
Should update this script to run `test:wdio`

Reply:
Yeah, lets get this updated, then I think this is read to merge down.